data/test_config/embedding_random_seed.yaml

```diff
@@ -1,3 +1,5 @@
 policies:
 - name: TEDPolicy
   random_seed: 42
+  validation_split: 0
+  max_history: 5
```

Comment:
let's remove the whole ymal. It's now the same as ted yaml

Reply:
Just wanted to test if it is related to the config. Seems like that `max_history` makes some kind of difference... The option `validation_split` does not exist anymore, if I am not mistaken. Strange. Any idea why this could be? But can remove the file from the test for now.

Reply:
Just wanted to test if it is related to the config. Seems like that max_history makes some kind of difference... The option validation_split does not exist anymore, if I am not mistaken. Strange. Any idea why this could be? But can remove the file from the test for now.

Reply:
ok, it is still failing... will remove the file

Reply:
that's very strange, we have 2 identical configs, but one is failing and the other is not

Reply:
is there anything that we store between the two runs? a cache or something that is not cleared?

Reply:
I don't know